composer.json

```diff
@@ -22,7 +22,7 @@
         "zendframework/zend-diactoros": "^1.1",
         "zendframework/zend-expressive-router": "^1.1",
         "zendframework/zend-expressive-template": "^1.0.1",
-        "zendframework/zend-stratigility": ">=1.1.0 < 1.3.0 || >=1.3.1 <2.0.0"
+        "zendframework/zend-stratigility": ">=1.2.0 < 1.3.0 || >=1.3.1 <2.0.0"
```

Comment:
Do we want still allow stratigility `1.2.0` here? Why not just from the latest version 1.3 (`1.3.3`)?
If still we want support `1.2` then maybe better starting from `1.2.1` with some fixes in compare to `1.2.0`?